rfcs/text/0018_timeslider.md

```diff
@@ -0,0 +1,217 @@
+
+Maps can then just mask data on the map based on some filter-expression. The evaluation of this filter-expression is done by mapbox-gl is fast because it occurs on the GPU. There is immediate visual feedback to the user as they manipulate the timeslider, because it does not require a roundtrip to update the data.
+
+##### 2.3.3.2 Caching of aggregated searches
+
+Aggregated data can be cached on the client, so toggling between timeslices can avoid a round-trip data-fetch. 
+The main technique here is for layers to use `.mvt`-data format to request data. Tiled-based data can be cached client-side
+
+We do _not_ propose _pre-fetching_ of aggregated data in this initial phase of the Maps timeslider effort. There is a couple reasons:
+- Based on the intended user-interactions for timeslider, because a user can flexible select a `timeslice` of arbitrary widths, it would be really hard to determine how to determine which timeslices to aggregate up front.
+- Maps already strains the maximum bucket sizes it can retrieve from Elasticsearch. Cluster/grid-layers often push up to 10k or more buckets, and terms-aggregations for choropleth maps also is going up to 10k buckets. Prefetching this for timeslices (e.g. say x10 timeslices) would easily exceed the default bucket limit sizes of Elasticsearch.
+
+
+##### 2.3.3.3 Decouple data-fetch from UI-effort
+
+Apart from refactoring the data-fetch for layers to now use two pieces of time-based state, the implementation will decouple any data-fetch considerations from the actual timeslider-UI work.
+ 
+The idea is that dial in data-fetch optimizations can be dialed into Maps in a parallel work-thread, not necessarily dependent on any changes or additions to the UI. 
+Any optimizations would not only affect timeslider users, but support all interaction patterns that require smooth data-updates (e.g. panning back&forth to two locations, toggling back&forth between two filters, ...)
+
+The main effort to support efficient data-fetch in a maps-context is to use `.mvt` as the default data format ([https://github.com/elastic/kibana/issues/79868](https://github.com/elastic/kibana/issues/79868)). This is a stack-wide effort in collaboration with the Elasticsearch team ([https://github.com/elastic/elasticsearch/issues/58696](https://github.com/elastic/elasticsearch/issues/58696), which will add `.mvt` as a core response format to Elasticsearch.
+
+Growing the use of `mvt` in Maps will help with both pre-fetching and client-side caching:
+- mvt is a binary format which allows more data can be packed inside a response, as compared to Json. Multiple tiles are patched together, so this introduces a form of parallelization as well. Due to growing the amount of data inside a single tile, and due to the parallelization, Maps has a pathway to increase the number of features that can be time-filtered.
```

Comment:
Change "mvt is a binary format which allows more data can be packed inside" to "mvt is a binary format which allows more data to be packed inside"